admin-ui/src/assets/i18n/ara.json

```diff
@@ -296,6 +296,51 @@
         "statuscheckFailed":" فشل",
         "statuscheckCompleted":"منجز"
     },
+    "packet-upload":{
```

Comment:
dont we download these files from artifactory?

Reply:
yes sir, but in here these files are used for testing it locally.